spec/uploader_spec.rb

```diff
@@ -22,6 +22,11 @@
     expect(result["width"]).to eq(TEST_IMG_W)
   end
 
+  it "should successfully upload a file from IO" do
+    result = Cloudinary::Uploader.upload(File.open(TEST_IMG, "rb"), :tags => [TEST_TAG, TIMESTAMP_TAG])
```

Comment:
It's not a good idea to open a file and never close it. Instead, try something like:

```suggestion
    File.open do |file|
      result = Cloudinary::Uploader.upload(file, :tags => [TEST_TAG, TIMESTAMP_TAG])
    end
```